### PR TITLE
feat(readAloud): landing Page

### PR DESCRIPTION
**Pull request by ovbm**
Screenshot 2022-08-30 at 15 21 41

Todo: 
- [x] Copy
- [x] Test if removal of buttonWidth didn't cause mayhem

### CODE REVIEW

**Comment by ovbm** [comment]
Tested ButtonWidth: Removal of fixed size doesn't change layout if no custom buttonLabel is set.

File: apps/www/components/Auth/EmailForm.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ const styles = {
     flexGrow: 1,
   }),
   button: css({
-    width: 160,
```
Comment (ovbm) [inline apps/www/components/Auth/EmailForm.js:28]: Fyi: tested and doesn't change layout if no custom buttonLabel is provided. Removing it allows us to add longer button labels to the Email from: eg. "Benachrichtigen"

Comment (trm217) [reply apps/www/components/Auth/EmailForm.js:28]: Are you sure there are no side-effects? If we just change it to minWidth we can ensure that the behaviour remains the same for short text but still allow for longer text.

**Review by trm217 — CHANGES_REQUESTED** [review]
See The "you're subscribed" label seems to be broken in all browsers (screenshot below). I guess a 'display: inline-block' must be missing in the element wrapping the checkbox + label

image

File: apps/www/components/Auth/EmailForm.js (hunk 1 above)
Comment (ovbm) [reply apps/www/components/Auth/EmailForm.js:28]: I am pretty sure. minWdith would also prevent shorter labels to look consistent

**Comment by patrickvenetz** [comment]
(The Vollständigkeit halber:) `inline` due to Editorial leads which are preceeded by a subject line

Bildschirmfoto 2022-09-02 um 14 16 05

**Comment by trm217** [comment]
Once again our "component library" is in no way predictable…

File: apps/www/components/Auth/EmailForm.js (hunk 1 above)
Comment (trm217) [reply apps/www/components/Auth/EmailForm.js:28]: Well until now the width was set to 160 fixed, so for labels that take up less space, the buttons would look different with this change.

**Review by trm217 — APPROVED** [review]

**Pull request merged**